### PR TITLE
Update names and dates for 2016.

### PR DESCRIPTION
**Pull request by parkr**
/cc @CUGC/nyyms for review.

### CODE REVIEW

File: app/views/nyym_mailer/signup_confirmation_to_attendee.text.erb [hunk 1]
```diff
@@ -2,15 +2,15 @@ Dear <%= @nyym.name %>,
 
 Thank you for registering for New York Young Men Sing with the Cornell University Glee Club. We are excited to sing with you! Here are the details of the event:
 
-Date: Saturday, February 7th 2015
+Date: Saturday, February 27th 2016
 Time: 10am - 5pm
 Place: Lincoln Hall and Sage Chapel at Cornell University
```
Comment (parkr) [inline app/views/nyym_mailer/signup_confirmation_to_attendee.text.erb]: @nyyms: is this still the location?

Comment (NYYMS) [inline app/views/nyym_mailer/signup_confirmation_to_attendee.text.erb]: Yup, the location is still Sage Chapel. I rolled back the time slightly to 5:30

**Comment by NYYMS** [comment]
@parkr When should the branch be merged into the main one? I looked over the edits and they all look good, is there any reason to not do it right now?

**Comment by parkr** [comment]
@nyyms did you change the length of the concert time everywhere?

**Comment by NYYMS** [comment]
@parkr I think so? I definitely did for the files that you had already worked on. How would I find any other places on the site/mailer that I might have missed? Is the search bar at the top the place to search for stuff like that?

**Comment by parkr** [comment]
Ok great, yes I see now. I'll deploy!

**Comment by parkr** [comment]
Deployed. You can see it now over at http://nyyms.gleeclub.com

**Pull request merged**